src/Rest/GlobalCommand.php

```diff
@@ -41,4 +43,38 @@ public function createApplicationCommand(
             ApplicationCommand::class
         );
     }
+
+    /**
+     * @see https://discord.com/developers/docs/interactions/application-commands#making-a-global-command
+     */
+    public function deleteApplicationCommand(
+        string $applicationId,
+        string $applicationCommandId
+    ): ExtendedPromiseInterface {
+        return new Promise(function (callable $resolve, callable $reject) use ($applicationCommandId, $applicationId) {
```

Comment:
Why is this wrapped in another promise? `$this->http->delete` already returns a promise. You're also never resolving said promise.